frontend/express/public/core/device-and-type/stylesheets/_main.scss

```diff
@@ -8,4 +8,12 @@
 
 .version-graph-block:last-child {
 margin-bottom: 12px;
+}
+
+.technologyDashboardWidget>.bu-column:nth-child(odd)>.cly-vue-section{
```

Comment:
Could you please replace camel case (technologyDashboardWidget) classes with kebab case (technology-dashboard-widget)? I mean, in order to maintain standards.